pkg/authorization/api/types.go

```diff
@@ -105,6 +105,7 @@ type RoleBinding struct {
 	RoleRef kapi.ObjectReference
 }
 
+// +genclient=true
```

Comment:
negative, but I should put extra new line after... similar to upstream: https://github.com/kubernetes/kubernetes/blob/master/pkg/api/v1/types.go#L344